src/sage/misc/dev_tools.py

```diff
@@ -203,7 +203,7 @@ def load_submodules(module=None, exclude_pattern=None):
             sys.stdout.write("failed\n")
 
 
-def find_objects_from_name(name, module_name=None):
+def find_objects_from_name(name, module_name=None, include_lazy_imports=False):
     r"""
```

Comment:
Some explanation about `include_lazy_imports` argument in the docstring?

Reply:
What is the use case of `include_lazy_imports=False`?

Reply:
The use case is interactive use of this function. 
Without this change in line 256, this function would pick up lazy imports from various modules. As in the modularization work, many more lazy imports are being introduced, this is becoming more noticeable.
It showed up in a doctest below, which is why I had to address it in this PR; but it's a regression in any case.

Reply:
Sorry I still don't get what you are saying. I am  confused because you introduced `include_lazy_imports=False` but it is not used anywhere, but still you say that you added this because of some doctest. Would you elaborate on what happens with what doctest?

The original `find_objects_from_name` is equivalent to `include_lazy_imports=True`. Right?

Reply:
OK. I now see what you meant is this one
```
        sage: import sympy
        sage: dt.find_objects_from_name('RR')
        [Real Field with 53 bits of precision, RR]   
```
which would pick up lazy-imported objects with `include_lazy_imports=True` (the original behavior).

Reply:
I've now added some an example to the docstring.

Reply:
Thanks.